src/data-table/index.js

```diff
@@ -98,13 +100,25 @@ class DataTable extends Component {
   }
 
   columns = () => {
-    const { children, columns } = this.props
+    const { children, columns, data, zeroConfig } = this.props
```

Comment:
* Instead of expecting a specific prop `zeroConfig`, auto apply zeroConfig when both `children` and `columns` is not present makes the usage of dataTable cleaner and more intuitive.
  All the user need to do is to have:
  ```
  <DataTable data={monsters} />
  ```
* I think we should move the zeroConfig chunk of code before line 105, as logic wise it's more important and should overwrite other props got pass in.
* need to make sure `data.length` is greater than one in this case, maybe handle the else case by giving error and returning a empty array.
* replacing `'_'` with `' '`  is nice and works for most of the time, but i think we should keep datatable as simple and straightforward as possible and give user the option to actually display `_` if they really want to. At the end, I think the user should take care of the data pre processing if they want to do so.